main.py

```diff
@@ -230,10 +245,8 @@ def create_model(self, name):
         model.setParam("MIPFocus", self.mip_focus)
         model.setParam("SolutionLimit", self.solution_limit)
         model.setParam("LogToConsole", self.log_to_console)
-
-        log_name = self.log_name if self.log_name else f"{self.problem} in mode {self.mode}"
-        model.setParam("LogFile", f"gurobi_logs/{datetime.date(datetime.now())}: "
-                                  f"{log_name}.log")
+        model.setParam("TimeLimit", 10000)
```

Comment:
Jeg ønsker å kjøre tester til og med 18 000 sekunder. Vil det påvirkes av denne?

Reply:
Yes this is a hard limit. If you want to be able to dynamically update it we have to add it to `__init__`. Should I do that?

Reply:
Ja, det hadde vært fint. Tror jeg er nødt til å kunne kjøre noen tester på 18 000, så fint om jeg har mulighet til å sette grensa selv for disse kjøringene.